app/docs/8-CSS.md

```diff
@@ -0,0 +1,23 @@
+# CSS
+
+On our project we are using [SCSS syntax](https://sass-scss.ru/guide/) and [CSS modules](https://github.com/css-modules/css-modules).
```

Comment:
https://sass-lang.com/guide
To be more consistent with other links (which are mainly lead to the English versions by default)